x-pack/plugins/security_solution/public/common/components/hover_actions/index.tsx

```diff
@@ -227,14 +232,18 @@ export const HoverActions: React.FC<Props> = React.memo(
       values,
     });
 
+    const Container = applyWidthAndPadding
```

Comment:
We wanted to use these actions in the overview cards where we don't want them to have a fixed width. Therefore I created two versions of it. One with and one without the extra width declarations. Alternatively I could have removed the `min-width` and `padding` altogether from this component to make it easier to integrate into other layouts. Essentially moving the responsibility to the parent component. If you feel like that is the way I should have done it, I'm happy to do it that way also.